### PR TITLE
Author Show Top Book Review

### LINKED ISSUE #29: User Story 15, Author Show Page displays top review for each book
```
As a Visitor,
When I visit an author's show page,
For each book written by that author
I should see one of the highest rated reviews
(review should contain the title, score, and user name)
```

DO THESE STORIES FIRST:
- User Story 14, Author Show Page
- User Story 13, Add a Review for a Book

### Checklist:

- [x] base code tests are written
- [x] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [x] code is reviewed
- [x] branch is merged
- [x] all tests still pass

child of #15

child of #33

depends on #14

depends on #16

### PR DESCRIPTION
**Pull request by BrennanAyers**
This PR brings in the functionality of the Author Show page to display the Top Review of all books they have authored. This was just a simple addition to the Book loop for our Authors, adding an if-block to determine whether the book has any reviews at all. If they do, it will take the top one, and print out the Title, Rating, and Username underneath each book.

Resolves #29

### CODE REVIEW

File: spec/features/authors/show_spec.rb [hunk 1]
```diff
@@ -54,5 +54,21 @@
...
+      visit author_path(@flapjacks)
+
+      within("#book-#{@book_1.id}-info") do
+        expect(page).to have_content(@review_1.title)
+        expect(page).to have_content("#{@review_1.rating} Stars")
+        expect(page).to have_content("Posted by: #{@review_1.user.name}")
+      end
+    end
   end
 end
```
Comment (alexander-mathieu) [inline spec/features/authors/show_spec.rb:74]: SOLID PASS! 👍
